charts_preview/actions-runner-controller-2/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: actions-runner-controller-2
+description: A Helm chart for install actions-runner-controller CRD
```

Comment:
We will be introducing the new OCI enabled chart publishing workflows soon, maybe it's best to avoid this folder structure change just for the sake of not triggering the existing workflows.

We can update existing release workflow easily to ignore `actions-runner-controller-2`: https://github.com/actions/actions-runner-controller/blob/3e88ae2d38b16c972c5af4fb448618767fa8d610/.github/workflows/publish-chart.yaml#L10